server/controllers/notification.js

```diff
@@ -0,0 +1,58 @@
+const Notification = require("../models/Notifications");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /notifications/create
+// @desc create new notification
+// @access Public
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const notification = new Notification(({ type, title, description, read, date, image, recipient } = req.body));
```

Comment:
Optional: It is a good practice to validate the request body before creating a new notification :)